apps/adk-orchestrator/test_orchestrator.py

```
@@ -0,0 +1,103 @@
+        
+        print("\n🎉 All tests passed!")
+        return True
+        
+    except Exception as e:
+        print(f"❌ Test failed: {e}")
+        return False
+
+def start_server():
+    """Start the orchestrator server"""
+    print("🚀 Starting orchestrator server...")
+    process = subprocess.Popen(
```

Ruff (0.12.2)

`subprocess` call: check for execution of untrusted input

(S603)